### ISSUE TITLE
Need to have warranty time set on trade-ins. 

### ISSUE THREAD

**Issue by kmcorp27** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1808]
We warranty our devices here and if someone comes back in for any refurbishment we dont have times set on the computer to allow our employee to know if it is within warranty period or not like our repairs do.

**Comment by Repairdesk-QA** [comment]
@ProductTeam-RepairDesk FYI

**Comment by jphansendk** [comment]
YES please... Was on my "hit RepairDesk in the head with this" list ;-)
